<launch>
  <param name="use_sim_time" value="true"/>
  <node pkg="rviz" type="rviz" name="rviz"
    args="-d $(find ar_pose)/demo/demo_single.rviz"/>
  <node pkg="tf" type="static_transform_publisher" name="world_to_cam" 
    args="0 0 0.5 -1.57 0 -1.57 world camera 1" />
  <node name="ar_pose" pkg="ar_pose" type="ar_single"
    respawn="false" output="screen">
    <param name="marker_pattern" type="string"
      value="$(find ar_pose)/data/4x4/4x4_1.patt"/>
    <param name="marker_width" type="double" value="152.4"/>
    <param name="marker_center_x" type="double" value="0.0"/>
    <param name="marker_center_y" type="double" value="0.0"/>
    <param name="threshold" type="int" value="100"/>
    <param name="use_history" type="bool" value="true"/>
  </node>
  <node pkg="rosbag" type="play" name="rosbag" 
    args="-l $(find ar_pose)/demo/ar_data/demo_single.bag --clock -d 3"/>
</launch>
